inst/Upgrades.qml

```diff
@@ -58,4 +58,858 @@ Upgrades
 			}
 		}
 	}
+
+	Upgrade
+	{
+		functionName: "LatentGrowthCurve";	fromVersion: "0.15";	toVersion: "0.16";
```

Comment:
can you change the version in `DESCRIPTION` and `inst/Description.qml` to `0.16.4`? Then use the `fromVersion: 0.16.3` and `toVersion: 0.16.4`. Also, perhaps put those on separate lines for better readability :)